### ISSUE TITLE
Bug - sev-2 - ENG: Update Letters App Headings

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7838]
The following updates need to be made in the VA letters and documents area of the app. 
_This list has been pulled in from the [full list of audit results](https://docs.google.com/spreadsheets/d/1KnR9Isdw8e3aBp5WhOFk7vaR4p6v4-wpYIO4GsEk4Go/edit?usp=sharing)._

- [ ] **Content:** Branch of service
     - **Location:** Benefits / VA letters and documents / Review letters / Letter details
     - **Currently a heading?:** No
     - **Recommended change:** We might want to consider making this a heading.
- [ ] **Content:** Discharge type
     - **Location:** Benefits / VA letters and documents / Review letters / Letter details
     - **Currently a heading?:** No
     - **Recommended change:** We might want to consider making this a heading.
- [ ] **Content:** Active duty start
     - **Location:** Benefits / VA letters and documents / Review letters / Letter details
     - **Currently a heading?:** No
     - **Recommended change:** We might want to consider making this a heading.
- [ ] **Content:** Separation date
     - **Location:** Benefits / VA letters and documents / Review letters / Letter details
     - **Currently a heading?:** No
     - **Recommended change:** We might want to consider making this a heading.

**Comment by brea11y** [comment]
Ticket has been reviewed and is still valid.

**Comment by alexandec** [comment]
@brea11y this one's a bit unusual because the way the items are announced currently, the bold text and the regular text are announced and focused together in one chunk. Like:

**Branch of service**
Coast guard
(announced together as "Branch of service coast guard, 1 of 4")
Swipe right ->

**Discharge type**
Honorable
(announced together as "Discharge type honorable, 2 of 4")

Here's a video of it:

[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1VlQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--89e8c0eddb1cebfc5e87c9a73f6e50c83e64f5a8/letter-details-annoncements.webm)

Normally from what I've seen, a header would be focused as its own separate item and announced separately. For example "Branch of service, header" would get focus and be announced. Then they'd swipe right, and "Coast guard" would get focus and be announced. Making the Letter Details screen work that way would require more swipes for them to work through all the items, and our "1 of 4" announcement wouldn't make sense anymore.

Just trying to figure out how (or if) things should be modified in this case. What do you think?

**Comment by brea11y** [comment]
Ahhh, thank you, @alexandec. You are exactly right. I was looking at them more in terms of styling and the groupings and didn't consider that they were reading out the additional copy. I'd say leave those as-is then. If we ever make a change to the way that the content is delivered in the future, we can update them then.

**Comment by alexandec** [comment]
@brea11y that makes sense to me, I'll close this ticket with no changes.

**Comment by brea11y** [comment]
Thanks so much!
